tonic_validate/metrics/context_length_metric.py

```diff
@@ -18,7 +18,8 @@ def __init__(
         max_length: Optional[int] = None,
     ):
         """
-        Create a metric that checks if each item in the context list is within a given length.
+        Create a binary metric that checks if each item in the context list is within a given length.
+        Returns 1 (True) if the length is within the given range, 0 (False) otherwise.
```

Comment:
Returns 1 (True) if the length is within the given length range. Returns 0 (False) if the context length falls outside of the range.